genome_kit/variant_genome.py

```diff
@@ -198,10 +200,34 @@ def find_motif(self, interval, motif, match_position=0, find_overlapping_motifs=
     def __repr__(self):
         return '<VariantGenome {} with {} variants>'.format(self.reference_genome, len(self.variants))
 
+    @property
+    def reference_genome(self):
+        return self.genome.reference_genome
+
+    def __eq__(self, other):
+        if not isinstance(other, VariantGenome):
+            return False
+        if self.genome != other.genome:
+            return False
+        # sort and normalize both sides
```

Comment:
maybe add a comment that overlapping variants would cause a conflict anyways, so ok to be ambiguous on start